README.md

```diff
@@ -51,7 +51,7 @@ work with the Hub development environment.
 
 ### Contributing
 
-1. Fork the repo ( https://github.com/18F/hub/fork )
+1.  Fork the repo ( https://github.com/18F/hub/fork ). If you're an 18F team member, you'll likely find it easier to clone the repo instead of forking it (`git clone --recursive git@github.com:18F/hub.git`). The recursive clone ensures that you'll grab the contents of the private data submodule.
```

Comment:
I'd say "private submodules" instead of "private data submodule". There's `_data/private` and `pages/private`.